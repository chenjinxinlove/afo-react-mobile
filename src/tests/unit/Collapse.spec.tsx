import * as React from 'react';
import * as renderer from 'react-test-renderer';
import DemoTag from '../../pageages/collapse/demo';

describe('Button', () => {
  // 测试快照
  it('test snapshots', () => {
    const component = renderer.create(
      <DemoTag />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
