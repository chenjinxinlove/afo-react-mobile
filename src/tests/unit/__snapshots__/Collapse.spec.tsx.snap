// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button test snapshots 1`] = `
<div>
  <p>
    基本用法
  </p>
  <div
    className="afo-collapse afo-hairline--top-bottom"
  >
    <div
      className="afo-collapse-item afo-collapse-item--expanded"
    >
      <div
        className="afo-cell afo-cell--clickable afo-hairlines afo-collapse-item__title"
        onClick={[Function]}
      >
        <div
          className="afo-cell__title"
        >
          <span>
            1
          </span>
        </div>
        <i
          className="afo-icon afo-icon-arrow afo-cell__right-icon"
        />
      </div>
      <div
        className="afo-collapse-item__wrapper"
        onTransitionEnd={[Function]}
      >
        <div
          className="afo-collapse-item__content"
        >
          111111111
        </div>
      </div>
    </div>
    <div
      className="afo-collapse-item afo-hairline--top"
    >
      <div
        className="afo-cell afo-cell--clickable afo-hairlines afo-collapse-item__title"
        onClick={[Function]}
      >
        <div
          className="afo-cell__title"
        >
          <span>
            2
          </span>
        </div>
        <i
          className="afo-icon afo-icon-arrow afo-cell__right-icon"
        />
      </div>
      
    </div>
    <div
      className="afo-collapse-item afo-hairline--top"
    >
      <div
        className="afo-cell afo-cell--clickable afo-hairlines afo-collapse-item__title"
        onClick={[Function]}
      >
        <div
          className="afo-cell__title"
        >
          <span>
            3
          </span>
        </div>
        <i
          className="afo-icon afo-icon-arrow afo-cell__right-icon"
        />
      </div>
      
    </div>
  </div>
  <p>
    手风琴
  </p>
  <div
    className="afo-collapse afo-hairline--top-bottom"
  >
    <div
      className="afo-collapse-item"
    >
      <div
        className="afo-cell afo-cell--clickable afo-hairlines afo-collapse-item__title"
        onClick={[Function]}
      >
        <div
          className="afo-cell__title"
        >
          <span>
            1
          </span>
        </div>
        <i
          className="afo-icon afo-icon-arrow afo-cell__right-icon"
        />
      </div>
      
    </div>
    <div
      className="afo-collapse-item afo-hairline--top"
    >
      <div
        className="afo-cell afo-cell--clickable afo-hairlines afo-collapse-item__title"
        onClick={[Function]}
      >
        <div
          className="afo-cell__title"
        >
          <span>
            2
          </span>
        </div>
        <i
          className="afo-icon afo-icon-arrow afo-cell__right-icon"
        />
      </div>
      
    </div>
    <div
      className="afo-collapse-item afo-hairline--top"
    >
      <div
        className="afo-cell afo-cell--clickable afo-hairlines afo-collapse-item__title"
        onClick={[Function]}
      >
        <div
          className="afo-cell__title"
        >
          <span>
            3
          </span>
        </div>
        <i
          className="afo-icon afo-icon-arrow afo-cell__right-icon"
        />
      </div>
      
    </div>
  </div>
</div>
`;
